Library/Homebrew/dev-cmd/bump-formula-pr.rb

```diff
@@ -83,6 +114,10 @@ def bump_formula_pr
       checked_for_duplicates = true
     end
 
+    if OS.linux? && formula.tap.core_tap?
+      tap, origin_branch, previous_branch = handle_linux(formula)
```

Comment:
```suggestion
      tap, origin_branch, previous_branch = use_correct_linux_tap(formula)
    else
      tap = formula.tap.full_name
      origin_branch = "origin/master"
      previous_branch = "-"
```

If these can't be `nil` from `use_correct_linux_tap`/`handle_linux`.

If they can maybe doing the `||=` directly below would be nicer.

Reply:
I've put an early return in the function itself with default values.